scripts/Info.plist

```diff
@@ -9,7 +9,7 @@
     <key>CFBundleInfoDictionaryVersion</key>
     <string>6.0</string>
     <key>CFBundlePackageType</key>
-    <string>dSYM</string>
+    <string>FMWK</string>
```

Comment:
One more reason to implement this in CMake. Usually this file can be tuned by user's variables. Just thought aloud.